app/src/main/res/drawable/ic_baseline_keyboard_arrow_down_24.xml

```diff
@@ -0,0 +1,10 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
it may be a duplicate of arrow_down and icon_down
left on with a name ic_icon_down